packages/dashboard/src/components/propertiesPanel/propertiesPanel.tsx

```diff
@@ -13,11 +17,24 @@ export interface PropertiesPanelProps {
 /** Panel element responsible for rendering chart configuration sections. */
 export function PropertiesPanel({ sections }: PropertiesPanelProps) {
   const selection = useSelection();
+  const dispatch = useDispatch();
+  const rightPaneCollapsed = useSelector((state: DashboardState) => state.rightPaneCollapsed);
 
   if (selection == null) {
+    if (rightPaneCollapsed) {
+      return <PropertiesPanelCollapsed />;
```

Comment:
I think it would be useful to create a collapsible pane component that would work for both the resource explorer and properties panel instead of making 2 unique components.

Reply:
done, using a single component for both panels